Libraries/LibWeb/XML/XMLDocumentBuilder.cpp

```diff
@@ -99,7 +99,23 @@ void XMLDocumentBuilder::element_start(const XML::Name& name, HashMap<XML::Name,
         m_namespace_stack.last().depth += 1;
     }
 
-    auto node = DOM::create_element(m_document, MUST(FlyString::from_deprecated_fly_string(name)), m_namespace).release_value_but_fixme_should_propagate_errors();
+    auto qualified_name_or_err = DOM::validate_and_extract(m_document->realm(), m_namespace, MUST(FlyString::from_deprecated_fly_string(name)));
```

Comment:
We use unabbreviated variable names wherever possible, so `qualified_name_or_error` would be better in this case. The same applies to `node_or_err` below.

Reply:
Thanks for the insight, I pushed a new version with updated variable names